src/equations/shallow_water_wet_dry_1d.jl

```diff
@@ -427,21 +470,35 @@ and for curvilinear 2D case in the paper:
 """
 @inline function Trixi.flux_nonconservative_fjordholm_etal(u_ll, u_rr,
                                                            orientation::Integer,
-                                                           equations::ShallowWaterEquationsWetDry1D)
-    return Trixi.flux_nonconservative_fjordholm_etal(u_ll, u_rr, orientation,
-                                                     equations.basic_swe)
+                                                           equations::ShallowWaterEquations1D)
+    # Pull the necessary left and right state information
+    h_ll, _, b_ll = u_ll
+    h_rr, _, b_rr = u_rr
+
+    h_average = 0.5f0 * (h_ll + h_rr)
+    b_jump = b_rr - b_ll
+
+    # Includes two parts:
+    #  (i)  Diagonal (consistent) term from the volume flux that uses `b_ll` to avoid
+    #       cross-averaging across a discontinuous bottom topography
+    #  (ii) True surface part that uses `h_average` and `b_jump` to handle discontinuous bathymetry
```

Comment:
```suggestion
    #  Surface contribution that uses `h_average` and `b_jump` to handle discontinuous bathymetry
```
This is an old comment that should have been updated when we restructured how the nonconservative fluxes are computed.